Apps/Examples/Examples/All Examples/OfflineRegionManagerExample.swift

```diff
@@ -44,7 +44,8 @@ public class OfflineRegionManagerExample: UIViewController, ExampleProtocol {
     }
 
     internal func setupExample() {
-        let offlineRegionDef = OfflineRegionGeometryDefinition(styleURL: mapView.style.uri.rawValue,
+        let uriString = mapView.style.uri!.rawValue
```

Comment:
nit: rename to `urlString` and the function takes a `styleURL`

Reply:
it is a uri though..

Reply:
aaaa can we rename the `OfflineRegionGeometryDefinition` to take a `styleURI` then. its confusing.

Reply:
Let's not worry - these are deprecated APIs 😄